js/src/components/google-combo-account-card/connect-ads/connect-ads-footer.js

```diff
@@ -14,14 +15,14 @@ import DisconnectAccount from '.~/components/google-ads-account-card/disconnect-
  *
  * @return {JSX.Element} Footer component.
  */
-const ConnectAdsFooter = ( { isConnected } ) => {
+const ConnectAdsFooter = ( { isConnected, onCreateNew = noop } ) => {
```

Comment:
```suggestion
const ConnectAdsFooter = ( { isConnected, onCreateNew } ) => {
```
From the code, looks like `onCreateNew` will always be defined.